cypress.json

```diff
@@ -1,3 +1,5 @@
 {
-  "baseUrl": "http://localhost:3000"
+  "env":{
```

Comment:
Taki szczegół, który pochłonął nam sporo czasu. Super, że to znalazłeś @Dimonium-239 :clap: